Makefile

```diff
@@ -101,16 +101,16 @@ test:
 	cp -r * .staging/test
 	-rm .staging/test/Dockerfile
 	cp test/Dockerfile .staging/test/Dockerfile
-	docker build --pull .staging/test -t gatekeeper-test && docker run -t gatekeeper-test
+	docker build --pull .staging/test -t gatekeeper-test && docker run -it gatekeeper-test
```

Comment:
Do we want to be running an interactive session here?

Reply:
Without this flag, I can't cancel a run of `make test` in my shell.  Yesterday, they were intermittently running slow and I found this very annoying.

Can remove if you feel it has a downside, I figured it was harmless and helpful in development.

Reply:
Good to know. If it doesn't interfere with the GitHub presubmit checks, SGTM.

Reply:
Presubmits are passing

Reply:
i think -it will break CI

Reply:
removed